homeassistant/helpers/state.py

```diff
@@ -202,11 +203,12 @@ def state_as_number(state):
 
     Raises ValueError if this is not possible.
     """
-    if state.state in (STATE_ON, STATE_LOCKED, STATE_ABOVE_HORIZON,
-                       STATE_OPEN):
+    if state.state in (STATE_ON, STATE_LOCKED, STATE_ABOVE_HORIZON, STATE_OPEN,
+                       STATE_HOME, STATE_PLAYING):
```

Comment:
This change of state type will create a type mismatch error for Influxdb schema.